hardware_interface/include/hardware_interface/actuator_handle.hpp

```diff
@@ -18,22 +18,14 @@
 #include <string>
 
 #include "hardware_interface/handle.hpp"
-#include "hardware_interface/macros.hpp"
-#include "hardware_interface/visibility_control.h"
 
 namespace hardware_interface
 {
 /** A handle used to get and set a value on a given actuator interface. */
-class ActuatorHandle : public Handle<ActuatorHandle>
+class ActuatorHandle : public ReadWriteHandle<ActuatorHandle>
```

Comment:
Yes this is for handling transmissions. We don't want `CommandHandle` to have any relation to `ActuatorHandle` but in the implementation.

Reply:
Maybe we should rename this so that the nomenclature do not collide with components.

"Joint" is OK because is not used anymore, we should check for the "Actuator"

Reply:
In a few PR-s time, they should be removed. Let's save time on that renaming ;)